mlflow/store/artifact/gcs_artifact_repo.py

```diff
@@ -88,13 +106,39 @@ def log_artifact(self, local_file, artifact_path=None):
         blob = gcs_bucket.blob(dest_path, chunk_size=self._GCS_UPLOAD_CHUNK_SIZE)
         blob.upload_from_filename(local_file, timeout=self._GCS_DEFAULT_TIMEOUT)
 
+    def _retryable_log_artifact(self, bucket, path, file_name):
+        gcs_bucket = self._get_bucket(bucket)
+        try:
+            gcs_bucket.blob(path, chunk_size=self._GCS_UPLOAD_CHUNK_SIZE).upload_from_filename(
+                file_name, timeout=self._GCS_DEFAULT_TIMEOUT
+            )
+        except Exception as e:
+            from google.cloud.storage import Client
+            from google.oauth2.credentials import Credentials
+
+            _logger.info(
+                "Failed to complete upload request, possibly due to credential expiration."
+                f" Attempting to refresh credentials and trying again... (Error: {e})"
+            )
+            if self.credential_refresh_def is not None:
+                new_token = self.credential_refresh_def()
```

Comment:
```suggestion

            if self.credential_refresh_def is not None:
                _logger.info(
                "Failed to complete upload request, possibly due to credential expiration."
                f" Attempting to refresh credentials and trying again... (Error: {e})"
            )
                new_token = self.credential_refresh_def()
```

Reply:
This logger message is to indicate that we failed due to a potential credential expiration and that we are attempting to refresh again.  I don't think it should be in the if block.